.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+tasks:
+  - init: make build
+github:
+  prebuilds:
+    master: true
```

Comment:
Does this need to be `main`? Or is `main` really mean default branch?

Reply:
`master` means `default` branch here (for historic reasons)